generator/_includes/nlg-description.md

```diff
@@ -0,0 +1,3 @@
+The Mass Defense Committee (MDC) is a network of lawyers, legal workers and law students providing legal support for political activists, protesters and movements for social change.
```

Comment:
This describes MDC, but the file is title `nlg-description` - should that be chanced?

Reply:
The MDC is a subset of the NLG. I guess we could name this file `mdc-description` but mdc means less to me than nlg 🤷‍♀️